### ISSUE TITLE
Consistent emergency evaluator opening

### ISSUE THREAD

**Issue by kasperosterbye** [guillep/pharo-core#5570]
I had a most surprising emergency evaluator pop up on me.

This code consistently opens the emergency evaluator:

```smalltalk
Deprecation new
		context: thisContext;
		explanation: 'shit happened';
		date: '2019-01-24';
		version: #Pharo8;
		signal.
```

I have been able to reproduce it from Playground in:

* Pharo9.0.0 build: 17, commit: cd8a657
* Pharo 8.0.0 build: 1121, commit: bbcdf97 and onwards

It was OK in Pharo 8.0.0 uptil and including build: 1120, commit: c115bf4

I am doing this on a Macbook Pro, Catalina, 64 bit.

<img width="1038" alt="emergency" src="https://user-images.githubusercontent.com/17723745/73064164-da841480-3ea0-11ea-9f7b-afbccb97e7db.png">

**Comment by JanBliznicenko** [comment]
Current build is 1124 which has it fixed. See issue #5545. The fix has not been integrated into Pharo 9 branch yet.

**Comment by estebanlm** [comment]
yeah, a problem in our latest integration of Spec2 (I should learn to not do that one day before release).
Anyway this is fixed upstream in 53c39353246e1cb84ae232709f26b152a984fd68
